network-api/networkapi/wagtailpages/static/js/directory-listing-filters.js

```diff
@@ -51,9 +51,20 @@
    */
   function loadResults(profiles) {
     let cards = profiles.map(profile => {
+      let { location } = profile;
+      let metaLocation = "";
+      let cardClass = "person-card";
+
+      if (location) {
+        metaLocation = `<p class="d-flex align-items-center meta-block-location h6-heading my-2">
+              ${profile.location}
+            </p>`;
+      } else {
+        cardClass = "person-card-sans-location";
```

Comment:
Why the rewrite instead of an overide? E.g. `class="person-card"` => `class="person-card no-card-location"`?